glacier/src/actions/remove-field.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type RemoveFieldAction = ReduxStandardAction<"REMOVE_FIELD", Field[]>;
+
+export function createRemoveFieldAction(fields: Field[]): RemoveFieldAction {
```

Comment:
Same comment as on add: Either pluralize the name or only take a single field as an argument.